lib/index.js

```diff
@@ -437,7 +437,7 @@ module.exports.renderSync = function(opts) {
     return result;
   }
 
-  throw util._extend(new Error(), JSON.parse(result.error));
+  throw assign(new Error(), JSON.parse(result.error));
```

Comment:
Maybe @am11 can shed some light on why this approach was taken in https://github.com/sass/node-sass/commit/50f374dfd3639486011bf2a3efa4f3f0b78eb64a?